.ci/windows-steps.yaml

```diff
@@ -12,8 +12,8 @@ steps:
     nuget install boost -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_random-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_math_c99-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
-    nuget install OpenBLAS -o $(Agent.ToolsDirectory)
```

Comment:
Any particular reason to remove this?  Seems like we should keep it installed.

Reply:
It is a repeated line, it was first mentioned in line 10 or 11.

Reply:
Ahh, ok; I did not expand the lines to see that. :smile: